custom_components/mass/config_flow.py

```diff
@@ -215,7 +215,11 @@ async def async_step_manual(self, user_input: dict[str, Any] | None = None) -> F
 
         try:
             self.server_info = await get_server_info(self.hass, user_input[CONF_URL])
-            self.openai_agent_id = user_input[CONF_OPENAI_AGENT_ID]
+            self.openai_agent_id = (
+                user_input[CONF_OPENAI_AGENT_ID]
+                if user_input[CONF_OPENAI_AGENT_ID] is not None
+                else ""
+            )
```

Comment:
```suggestion
            self.openai_agent_id = user_input[CONF_OPENAI_AGENT_ID] or ""
```

this is more pythonic

Reply:
Done